apps/core0/subsys/containers/filesystem.go

```diff
@@ -126,8 +148,8 @@ func (c *container) sandbox() error {
 			c.cleanSandbox()
 		},
 	}
-
-	if err := c.mountFList(c.Args.Root, root, onSBExit); err != nil {
+	cfg := c.Args.Config
```

Comment:
just pass this directly to the mountFlist call, don't create a temporary variable